less/admin/UsersListPage.less

```diff
@@ -0,0 +1,127 @@
+.UserListPage {
+  // Pad bottom of page to make nav area look less squashed
+  padding-bottom: 24px;
+
+  &-grid {
+    width: 100%;
+    position: relative;
+    border-radius: @border-radius;
+
+    // Use CSS custom properties to define the number of columns in the grid
+    grid-template-columns: repeat(var(--columns), max-content);
+
+    // Ensure mobile scrollbar isn't on top of content
+    padding-bottom: 4px;
+
+    // Table refreshing overlay
+    &--loadingPage {
+      &::after {
+        content: "";
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background: rgba(128, 128, 128, 0.2);
+      }
+
+      .LoadingIndicator-container {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+      }
+    }
+
+    &--loaded,
+    &--loadingPage {
+      display: grid;
+      overflow-x: auto;
+    }
+
+    &--header {
+      font-weight: bold;
+      border-bottom: 1px solid @muted-more-color;
+      padding: 8px 16px;
+      background: @control-bg;
+    }
+
+    &--rowItem {
+      padding: 4px 16px;
+
+      &[data-column-name="editUser"] {
+        padding: 0;
+        position: relative;
+      }
+    }
+
+    &--shadedRow {
+      background: darken(@body-bg, 3%);
+
+      & when (@config-dark-mode = true) {
+        background: lighten(@body-bg, 5%);
+      }
+    }
+  }
+
+  &-gridPagination {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+}
+
+// Handles styling of default UserList columns
+.UserList {
+  &-joinDate {
+    cursor: help;
+    text-decoration: underline;
+    text-decoration-style: dotted;
+  }
+
+  &-editModalBtn {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    border-radius: 0;
+  }
+
+  &-email {
+    display: flex;
+
+    &[data-email-shown="false"] {
+      .UserList-emailAddress {
+        user-select: none;
+        filter: blur(4px);
+        cursor: pointer;
+      }
+    }
+
+    &Address {
+      flex-grow: 1;
+      margin-right: 4px;
+      transition: filter 0.2s ease-out;
+    }
+
+    &IconBtn {
+      appearance: none;
+      border: none;
+      padding: 2px;
+      margin: 0;
+      margin-left: 12px;
+      background: transparent;
+      outline: none;
+    }
```

Comment:
I prefer using `Button Button--text` and dropping what can be dropped from these declarations.

Reply:
I find this easier because I'd need to override the background and everything else from the default button, and it's not actually a textual button either (hence the modifier doesn't really make any sense). I initially tried with the background there, but it felt far too clunky and didn't look nice.

Reply:
I wasn't suggesting to change the design (I [realize](https://github.com/flarum/core/blob/43d6b3104d8ca7d4343aa864dd898e2af38f955a/less/common/Button.less#L153-L167) not everyone feels the same about it)

> because I'd need to override the background and everything else from the default button

That's the thing, you wouldn't, `Button--text` is a class for a style stripped button, you're already overriding things here, it would more consistent to use the built in generic classes we have for situations like these, and not have to override anything a second time, `Button Button-text` would preserve the same current look, while staying consistent, which is important for theming purposes 🤗

Reply:
Ohhh! I read it as `.Button-text` instead so assumed it was something else -- that's super helpful, actually!